test/dummy/app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
probably unnecessary. The app running ish_manager is microsites3 (different repo). test/dummy is used for testing ONLY. test/dummy is not a real application